opencog/guile/README

```diff
@@ -100,6 +100,18 @@ utilities can be pulled up at the scheme shell prompt, by saying
 have to consult the individual scm files for more info.
 
 
+== Modules ==
+-------------
+Currently OpenCog's scheme functions are broken into 3 different scheme
+modules: (opencog), (opencog extension), and (opencog atomtypes).  Each
+one can be listed with the following code:
+
+(module-for-each (lambda (symbol value) (display symbol) (newline))
+(resolve-interface '(opencog)))
+
+Replace '(opencog) with the corresponding module name.
```

Comment:
This needs to be on the wiki too. 

Once upon a time, the wiki and the README were copies of each other, but have gotten out of sync ..